core/src/main/java/com/adobe/aem/commons/assetshare/content/AssetModel.java

```diff
@@ -62,4 +62,19 @@
      * @return a ValueMap composed of a look up based on: 1) ComputedProperties, the [dam:Asset]/jcr:content/metadata ValueMap and finally the [dam:Asset] ValueMap.
      */
     public ValueMap getProperties();
+
+    /**
+     * Returns true if the asset is a Content Fragment
+     *
+     * @return true if the asset is a Content Fragment
+     */
+    boolean isContentFragment();
```

Comment:
We shouldnt add these methods to the AssetModel interface, instead we should leverage computed properties..

ie. `${asset['assetType'] == 'CONTENT-FRAGMENT'}`

Or if you really want something that returns a boolean, then a new computed property .. `${asset['isContentFragment']}` .. tho this feels overkill?